### PR TITLE
Updated to use formats from new and hopefully more reliable source

### PR DESCRIPTION
**Pull request by stufflebear**
So now we're using the formats from http://www.deverusdemos.com/Help/FlashHelp/Search_Types/Formats_for_MVR_license_numbers.htm

Why? Because this is what our background checking software uses and the other document was flat out wrong about some states.

r? @samstr @bordicon

### CODE REVIEW

**Comment by samstr** [comment]
looks cool to me. @bordicon plz check

are we strict about adding to `CHANGES.txt`?

File: dlnvalidation.py [hunk 1]
```diff
@@ -3,57 +3,57 @@
 
 # In alphabetical order by state, not by abbreviation
 STATE_FORMATS = {
-    'AL': r'[0-9]{1,7}$',
-    'AK': r'[0-9]{1,7}$',
-    'AZ': r'[a-z][0-9]{1,8}$|[a-z]{2}[0-9]{2,5}$|[0-9]{9}$',
-    'AR': r'[0-9]{4,9}$',
+    'AL': r'[0-9]{7}$|[a-z0-9]{6}$',
```
Comment (samstr) [inline dlnvalidation.py]: Start all these expressions with `^` ?

**Comment by stufflebear** [comment]
Fixed comments

**Comment by samstr** [comment]
r+

**Pull request merged**